clang/test/Driver/clang-offload-deps.c

```diff
@@ -0,0 +1,40 @@
+// REQUIRES: x86-registered-target
+
+//
+// Check help message.
+//
+// RUN: clang-offload-deps --help | FileCheck %s --check-prefix CHECK-HELP
+// CHECK-HELP: {{.*}}OVERVIEW: A tool for creating dependence bitcode files for offload targets. Takes
+// CHECK-HELP: {{.*}}host image as input and produces bitcode files, one per offload target, with
+// CHECK-HELP: {{.*}}references to symbols that must be defined in target images.
+// CHECK-HELP: {{.*}}USAGE: clang-offload-deps [options] <input file>
+// CHECK-HELP: {{.*}}  --outputs=<string> - [<output file>,...]
+// CHECK-HELP: {{.*}}  --targets=<string> - [<offload kind>-<target triple>,...]
```

Comment:
It won't work because this test does not check the full output. Here is how it looks like

OVERVIEW: A tool for creating dependence bitcode files for offload targets. Takes
host image as input and produces bitcode files, one per offload target, with
references to symbols that must be defined in target images.

USAGE: clang-offload-deps [options] <input file>

OPTIONS:

Generic Options:

  --help             - Display available options (--help-hidden for more)
  --help-list        - Display list of available options (--help-list-hidden for more)
  --version          - Display the version of this program

clang-offload-deps options:

  --outputs=<string> - [<output file>,...]
  --targets=<string> - [<offload kind>-<target triple>,...]

Reply:
I see. Probably we should check the test in OVERVIEW section with `CHECK-HELP-NEXT` and keep other checks as is, if we don't want to check the full output.

Reply:
Okay, I will add -NEXT to OVERVIEW section.